lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -863,6 +863,9 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
 
     # Perform the actual upload
     upload_csv_to_report_store(rows, 'grade_report', course_id, start_date)
+    TASK_LOG.info(u'Grade reported generated for course %s',
```

Comment:
I am going to remove this log as we can view the track record for generated grade reports via the instructor dashboard.